src/caselawclient/models/utilities/aws.py

```diff
@@ -58,8 +67,9 @@ def create_sns_client() -> SNSClient:
     return create_aws_client("sns")
 
 
-def uri_for_s3(uri: str) -> str:
-    return uri.lstrip("/")
+def uri_for_s3(uri: DocumentURIString) -> S3PrefixString:
```

Comment:
```suggestion
def uri_for_s3(uri: DocumentURIString) -> S3PrefixedString:
```

Reply:
Actually thinking about it this isn't prefixed at all, it's suffixed! Maybe something like `S3DocumentFolderString`? I know it's not _really_ a folder, but...

Reply:
I think it is a "prefix" itself.

Reply:
Agreed, I was approaching this from the wrong angle.